src/pages/Settings/SubscriptionsSettings.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-	<www-page class="subscriptions" :gray-background="true">
```

Comment:
I believe we want to keep all of the `:gray-background="true"` props intact

Reply:
Yes we do. Good call

Reply:
Good. Gray background changes are reverted now.